sql/localsql/nipost/nipost.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type NIPostState struct {
-	*types.NIPost
+	types.NIPost
```

Comment:
Now with the context of this PR I believe that `types.NIPost` should not be part of the domain type `types.ActivationTx` but ONLY of the wire type. This also means that it does not belong here (no wire types in `sql` packages).

This doesn't mean that we can't have a separate type here that is identical to the current `types.NIPost` so we can encode it as bytes and store it in the DB, it just means that this type is ONLY used to preserve the `NIPost` and for nothing else - it is not included in `types.ActivationTx` and for encoding the "`wire.ActivatonTxV1`" has it's own (sub-)type.